tests/TestDataService.spec.ts

```diff
@@ -95,6 +95,17 @@ test('Data Service', async ({
     const rule = await TestDataService.createBasicRule({ description: 'Rule description' });
     expect(rule.description).toEqual('Rule description');
 
+    const parentProduct = await TestDataService.createBasicProduct();
+    const propertyGroups: PropertyGroup[] = [];
+    const propertyGroupColor = await TestDataService.createColorPropertyGroup();
+    const propertyGroupText = await TestDataService.createTextPropertyGroup();
+    propertyGroups.push(propertyGroupColor);
+    propertyGroups.push(propertyGroupText);
+
+    const variantProducts = await TestDataService.createVariantProducts(parentProduct, propertyGroups, { description: 'Variant description'});
```

Comment:
Why do you need to add a variant description? 🤔

Reply:
I check with that, if the override function works so that you can define some specific variant values